src/cdomains/intDomain.ml

```diff
@@ -552,7 +552,7 @@ struct
               top_of ik
         else if not cast && should_ignore_overflow ik then
           let tl, tu = BatOption.get @@ top_of ik in
-          Some (max tl x, min tu y)
+          Some (Ints_t.max tl x, Ints_t.min tu y)
```

Comment:
Are all these insertions fo `Ints_t` simply to clarify what is going on, or where they related to a bug?

Reply:
I didn't observe a bug, but just noticed that the previous ones were calling the polymorphic `Stdlib.max`, which internally uses polymorphic comparison instead of `Ints_t.compare`. Better safe than sorry with these on bigints.